yoda/src/main/java/life/jukebox/HandshakeClient.java

```diff
@@ -0,0 +1,69 @@
+package life.jukebox.yoda;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A simple client that requests a greeting from the {@link HelloWorldServer}.
+ */
+public class HandshakeClient {
```

Comment:
This class/file should be named YodaClient... but we'll most likely scratch this whole client in a week or so so I don't care if you check it in with the name HandshakeClient